src/api/client.tsx

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Since, it is next.js application we can write Next.js apis using route handlers. Please go through the below docs for reference - 
https://nextjs.org/docs/app/building-your-application/routing/route-handlers
https://nextjs.org/docs/pages/building-your-application/routing/api-routes

Reply:
fixed